packages/components/ovh-at-internet/package.json

```diff
@@ -34,6 +34,7 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-config' --include-dependencies -- yarn run dev:watch"
   },
   "dependencies": {
+    "@ovh-ux/manager-config": "^5.0.0 || ^6.0.0",
```

Comment:
**issue(blocking):** `@ovh-ux/manager-config` is private package whereas `@ovh-ux/ovh-at-internet` is not.